### ISSUE TITLE
Errors picked up in console but not shown inline

### ISSUE THREAD

**Issue by jimbateson** [kungfusheep/SublimeLinter-contrib-stylelint#20]
Hi,

After following all of the setup instructions, I opened a file that has an linting error that should be flagged and noticed that although the console suggested stylelint had found an error, there was nothing to suggest inline that the line had an error on it.

This is the output from the console:

```
SublimeLinter: stylelint output:
CssSyntaxError: c:\users\{username}\appdata\local\temp\SublimeLinter3-bateson.j\notify.css:2:1: Missed semicolon
    at Input.error (C:\Users\{username}\node_modules\postcss\lib\input.js:61:22)
    at Parser.checkMissedSemicolon (C:\Users\{username}\node_modules\postcss\lib\parser.js:495:26)
    at Parser.decl (C:\Users\{username}\node_modules\postcss\lib\parser.js:270:50)
    at Parser.word (C:\Users\{username}\node_modules\postcss\lib\parser.js:135:30)
    at Parser.loop (C:\Users\{username}\node_modules\postcss\lib\parser.js:60:26)
    at parse (C:\Users\{username}\node_modules\postcss\lib\parse.js:26:12)
    at new LazyResult (C:\Users\{username}\node_modules\postcss\lib\lazy-result.js:61:24)
    at Processor.process (C:\Users\{username}\node_modules\postcss\lib\processor.js:34:16)
    at lintString (C:\Users\{username}\node_modules\stylelint\dist\standalone.js:88:9)
    at C:\Users\{username}\node_modules\stylelint\dist\standalone.js:70:14 

```

I have placed the `.stylelintrc` config file in my home directory as well as installed `postcss` and`stylelint` in there (and in the project directory just in case as I wasn't too sure)

I was just wondering if you had seen an issue like this before or if I need to set anything in my SublimeLinter user settings?

Any help would be much appreciated,

Thanks
Jim

**Comment by Arkkimaagi** [comment]
I'm trying to solve the same problem and I have no idea what's causing this. I'll let you know if I get it solved.

**Comment by Arkkimaagi** [comment]
Check this issue: https://github.com/kungfusheep/SublimeLinter-contrib-stylelint/issues/25
